.github/workflows/sync_github_issues_to_azdo.yml

```diff
@@ -0,0 +1,24 @@
+name: Sync issue to Azure DevOps work item
+
+on:
+  issues:
+    types:
+      [opened, edited, deleted, closed, reopened, labeled, unlabeled, assigned]
+
+jobs:
+  alert:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: danhellem/github-actions-issue-to-work-item@master
+        env:
+          ado_token: "${{ secrets.ADO_PERSONAL_ACCESS_TOKEN }}"
```

Comment:
note: I have created a personal access token & added it under secrets

Reply:
I think it's preferable to get tokens created for this by the organization/project owners in GitHub and AzDO respectively. If we use a personal token you created, we have to rely on you in the future for maintaining it and keeping it from expiring or swap to a new token owned by someone else occasionally.

I know for some other workflows @zhindes requested an organizational level secret (such as [here](https://github.com/ni/grpc-device/blob/main/.github/workflows/build_cmake.yml#L228)). We should at least entertain that approach instead, which will result in an organizational secret available on all NI repositories: 
![image](https://user-images.githubusercontent.com/5367780/125641625-b89ce6f2-7be0-4f60-a8c9-13d89e091f55.png)

Reply:
If you'd like to get this in to test it before getting the new access tokens created, I'm OK with that as well.

Reply:
Yeah, I would recommend an org-wide secret that isn't a personal token. Ping one of the NI AzDO/GitHub admins to get that created and setup. I usually talk directly to David Corrigan for stuff like this, but he may direct you to a better method of reaching out.

Reply:
Sorry on the delay on this, had a busy week last week and want to dig into the security features of GitHub a bit before setting up a token with R/W access to AzDO work items. I'll try and do that in the next few days.

Reply:
Added the tokens. The AzDO one is scoped pretty narrowly to just that area path so if that changes we'll need to update it. That area path should really be an ID and not a string too but that's not something you all could fix - it's just not as nice if area paths get rearranged in DevCentral.